### ISSUE TITLE
Run Dependency Analysis 

### ISSUE THREAD

**Issue by ashdavies** [ashdavies/playground.ashdavies.dev#422]
[dependency-analysis-android-gradle-plugin](https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/wiki/Customizing-plugin-behavior#failure-conditions-and-filtering)
[gradle/github-dependency-graph-gradle-plugin](https://github.com/gradle/github-dependency-graph-gradle-plugin)

**Comment by ashdavies** [comment]
Duplicate of #366

**Comment by ashdavies** [comment]
Blocked by autonomousapps/dependency-analysis-gradle-plugin/issues/751

**Comment by ashdavies** [comment]
Dependency analysis is not currently supported for Kotlin Multiplatform autonomousapps/dependency-analysis-gradle-plugin/issues/228

**Comment by ashdavies** [comment]
Supercedes #366
